### PR TITLE
Blog Post: fiatjaf Nostr LTS

### PR DESCRIPTION
**Pull request by Arvin21M**
**Build Preview:**
- [/blog/fiatjaf-recieves-lts-grant](https://os-website-git-fiatjaf-lts-open-sats.vercel.app/blog/fiatjaf-recieves-lts-grant)

**Social Preview:**
41-fiatjaf

### CODE REVIEW

File: data/blog/fiatjaf-recieves-lts-grant.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+OpenSats is proud to announce long-term support for fiatjaf, the innovative developer behind nostr.
+
+Before nostr, fiatjaf's primary focus was on decentralized financial systems and tools for the Bitcoin Lightning Network, 
+having contributed to projects like [lntxbot](https://github.com/fiatjaf/lntxbot), [c-lightning](https://github.com/ElementsProject/lightning/commits?author=fiatjaf), 
+[lnbits](https://github.com/lnbits/lnbits/commits?author=fiatjaf), and creating [LNURL tools](https://github.com/fiatjaf/go-lnurl), to name a few.
+
+When he is not writing inflammatory posts about how everybody is wrong, he spends his time tirelessly enhancing the [nostr](https://nostr.com/) ecosystem he pioneered.
+
+This LTS grant will enable fiatjaf to continue to improve the functionality and resilience of nostr through several ongoing projects, such as
```
Comment (dergigi) [inline data/blog/fiatjaf-recieves-lts-grant.mdx:19]:
```suggestion
This LTS grant will enable fiatjaf to focus all his time and attention on nostr and his many ongoing projects, such as 
```

File: data/blog/fiatjaf-recieves-lts-grant.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+[lnbits](https://github.com/lnbits/lnbits/commits?author=fiatjaf), and creating [LNURL tools](https://github.com/fiatjaf/go-lnurl), to name a few.
+
+When he is not writing inflammatory posts about how everybody is wrong, he spends his time tirelessly enhancing the [nostr](https://nostr.com/) ecosystem he pioneered.
+
+This LTS grant will enable fiatjaf to continue to improve the functionality and resilience of nostr through several ongoing projects, such as 
+[nostr-tools](https://github.com/nbd-wtf/nostr-tools), [nos2x](https://github.com/fiatjaf/nos2x), [go-nostr](https://github.com/nbd-wtf/go-nostr), 
+[khatru](https://github.com/fiatjaf/khatru), [eventstore](https://github.com/fiatjaf/eventstore), [njump](https://njump.me/about), [loure](https://github.com/fiatjaf/loure), 
+[wikistr](https://github.com/fiatjaf/wikistr), [patch34](https://github.com/fiatjaf/patch34), [nak](https://github.com/fiatjaf/nak), [gitstr](https://github.com/fiatjaf/gitstr), 
+[song](https://git.fiatjaf.com/song), [mmm](https://git.fiatjaf.com/mmm), and [promenade](https://git.fiatjaf.com/promenade). These tools and frameworks are essential for the
```
Comment (dergigi) [inline data/blog/fiatjaf-recieves-lts-grant.mdx:23]:
```suggestion
[song](https://git.fiatjaf.com/song), [mmm](https://git.fiatjaf.com/mmm), and [promenade](https://git.fiatjaf.com/promenade). These tools and frameworks are aimed at improving the 
```

File: data/blog/fiatjaf-recieves-lts-grant.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+[nostr-tools](https://github.com/nbd-wtf/nostr-tools), [nos2x](https://github.com/fiatjaf/nos2x), [go-nostr](https://github.com/nbd-wtf/go-nostr), 
+[khatru](https://github.com/fiatjaf/khatru), [eventstore](https://github.com/fiatjaf/eventstore), [njump](https://njump.me/about), [loure](https://github.com/fiatjaf/loure), 
+[wikistr](https://github.com/fiatjaf/wikistr), [patch34](https://github.com/fiatjaf/patch34), [nak](https://github.com/fiatjaf/nak), [gitstr](https://github.com/fiatjaf/gitstr), 
+[song](https://git.fiatjaf.com/song), [mmm](https://git.fiatjaf.com/mmm), and [promenade](https://git.fiatjaf.com/promenade). These tools and frameworks are essential for the 
+development of nostr clients, building custom relays, and enhancing the overall nostr infrastructure​.
+
+Additionally, fiatjaf will begin several new initiatives aimed at enhancing user experience, bolstering network resilience, and offering competitive alternatives to existing solutions. 
+These projects include:
+
+-   NIP29 GTK4 desktop client
```
Comment (dergigi) [inline data/blog/fiatjaf-recieves-lts-grant.mdx:29]:
```suggestion
-   NIP-29 GTK4 desktop client
```

**Review by dergigi — CHANGES_REQUESTED** [review]
Some minor edits and suggestions.

**Comment by Arvin21M** [comment]
@dergigi these two commit checks are not passing:

[Commit 1](https://github.com/OpenSats/website/pull/293/commits/bb886648cd565dac6d401038491504b2ab5372c7)
[Commit 2](https://github.com/OpenSats/website/pull/293/commits/dd1a70627aebfe7304837681704c4ff8d50ed1c9)

The system is outling the issue with both to be:

> The following actions uses Node.js version which is deprecated and will be forced to run on node20

But when I look at the workflow file for the others that passed checks, they have the same identical file. I am a bit perplexed at this point on how to resolve the failed checks.

**Comment by dergigi** [comment]
@Arvin21M if you click on the little "x" you will see why:

> Canceled by a more recent commit in the same branch

So that's normal and there's nothing to worry about.

**Comment by Arvin21M** [comment]
Ah I see. Got it!

> if you click on the little "x" you will see why

**Comment by dergigi** [comment]
Live now:
- https://opensats.org/blog/fiatjaf-receives-lts-grant
- [nostr post](https://njump.me/nevent1qqsycef79crjaek3t72ajsv2nu3zfh5dzf9q6wlmyqwfw3xfnl8slfczypu8xwr40lp96ewdj2fef408wy70gd3carf9n6xu7hrnhq6whpglyeqdhgy)

**Pull request merged**